scripts/captain.sh

```
@@ -0,0 +1,208 @@
+#!/bin/sh
+
```

why not setting any fails fast or stuff like that ? 
-eu etc

---

it takes about 10 to 15 secs for this script to run. We do not want it to fast fail. We want to know all the failures and we will also be passing several versions at once. We want to know everything that fails, for all versions we pass as input to this script.